### PR TITLE
build(deps): bump fast-xml-parser from 4.4.0 to 4.4.1

### PR DESCRIPTION
**Pull request by afonsobarracha**

### CODE REVIEW

File: wrappers/react-native/yarn.lock [hunk 1]
```diff
@@ -5106,11 +4581,11 @@ resolve-from@^4.0.0:
   integrity sha512-pb/MYmXstAkysRFx8piNI1tGFNQIFA3vkE3Gq4EuA1dF6gHp/+vgZqsCGJapvy8N3Q+4o7FwvquPJcnZ7RYy4g==
 
 resolve@^1.14.2:
-  version "1.22.1"
-  resolved "https://registry.yarnpkg.com/resolve/-/resolve-1.22.1.tgz#27cb2ebb53f91abb49470a928bba7558066ac177"
-  integrity sha512-nBpuuYuY5jFsli/JIs1oldw6fOQCBioohqWZg/2hiaOybXOft4lonv85uDOKXdf8rhyK159cxU5cDcK/NKk8zw==
+  version "1.22.8"
```
Comment (thomas-pearson-mattr) [inline wrappers/react-native/yarn.lock:4584]: wonder why the downgrade?

File: wrappers/react-native/yarn.lock [hunk 2]
```diff
@@ -2219,17 +1779,12 @@ camelcase@^6.2.0:
   resolved "https://registry.yarnpkg.com/camelcase/-/camelcase-6.3.0.tgz#5685b95eb209ac9c0c177467778c9c84df58ba9a"
   integrity sha512-Gmy6FhYlCY7uOElZUSbxo2UCDH8owEk996gkbrpsgGtrJLM3J7jGxl9Ic7Qwwj4ivOE5AWZWRMecDdF7hqGjFA==
 
-caniuse-lite@^1.0.30001400:
-  version "1.0.30001402"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001402.tgz#aa29e1f47f5055b0d0c07696a67b8b08023d14c8"
-  integrity sha512-Mx4MlhXO5NwuvXGgVb+hg65HZ+bhUYsz8QtDGDo2QmaJS2GBX47Xfi2koL86lc8K+l+htXeTEB/Aeqvezoo6Ew==
+caniuse-lite@^1.0.30001646:
+  version "1.0.30001649"
```
Comment (thomas-pearson-mattr) [inline wrappers/react-native/yarn.lock:1783]: Thats a fun looking version 🤣

**Review by mate-from-mattr — APPROVED** [review]

File: wrappers/react-native/yarn.lock (hunk 1 above)
Comment (afonsobarracha) [reply wrappers/react-native/yarn.lock:4584]: It was an upgrade from 1.22.1 to 1.22.8

Comment (thomas-pearson-mattr) [reply wrappers/react-native/yarn.lock:4584]: Lol must have read it the wrong way around 😆

**Review by MargarytaKolomiyets — APPROVED** [review]

**Review by thomas-pearson-mattr — APPROVED** [review]

**Pull request merged**